perception/ground_segmentation/include/ground_segmentation/scan_ground_filter_nodelet.hpp

```diff
@@ -54,9 +54,13 @@ class ScanGroundFilterComponent : public pointcloud_preprocessor::Filter
     POINT_FOLLOW,
     UNKNOWN,
     VIRTUAL_GROUND,
+    OUT_OF_RANGE
   };
   struct PointRef
   {
+    float gama;         // angle in vertical
```

Comment:
```suggestion
    float gamma;         // angle in vertical
```

Reply:
I removed the global variable as unused https://github.com/autowarefoundation/autoware.universe/pull/1899/commits/31fb64e2fa25488e424f4ad3435c5879e4e13d6f